SsrEnv TS error when using the isolated Modules flag

Current export of SsrEnv does not take into consideration it is only a type as it must be considered when --isolatedModules flag is provided
```
node_modules/vite-plugin-ssr/ssrEnv.node.ts:5:10 - error TS1205: Re-exporting a type when the '--isolatedModules' flag is provided requires using 'export type'.

5 export { SsrEnv }
           ~~~~~~


Found 1 error.
```

--- a/src/ssrEnv.node.ts
+++ b/src/ssrEnv.node.ts
@@ -2,7 +2,7 @@ import { ViteDevServer } from 'vite'
 
 export { setSsrEnv }
 export { getSsrEnv }
-export { SsrEnv }
+export type { SsrEnv }
 
 type SsrEnv =
   | {

